.circleci/config.yml

```diff
@@ -139,7 +142,26 @@ jobs:
       - run:
           name: Run e2e file upload features test
           command: |
-            python -m scripts.run_e2e_tests --suite="fileUploadFeatures"
+            python -m scripts.run_e2e_tests --skip_install --skip-build --suite="fileUploadFeatures" --prod_env
+
+  e2e_navigation:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /home/circleci/
+      - run:
+          <<: *install_dependencies
+      - run:
+          name: Run e2e file upload features test
+          command: |
+            python -m scripts.run_e2e_tests --prod_env --skip_install --suite="navigation"
+      - persist_to_workspace:
+          root: /home/circleci/
+          paths:
+            - oppia/third_party/
+            - oppia/build/
+            - oppia/backend_prod_files/
```

Comment:
Are these the folders where the build files are placed?

Reply:
@vojtechjelinek @ankita240796 do you know?

(Note: Kevin is referring to the files that are built during the build.py run -- i.e. the minified/webpack files needed for production.)

@kevinlee12 this may also be helpful: https://github.com/oppia/oppia/wiki/Webpack

Reply:
Yes, build files are present in these folders as far as I know.

Reply:
Not sure if third party is required completely, only a subsection has build files: `third_party/generated` I think.

Reply:
Thanks @ankita240796, @vojtechjelinek!  

@kevinlee12 Do you have the info you need?

Reply:
Yup, that's all I need, thanks everyone!